appengine/storage/standard/README.md

```diff
@@ -0,0 +1,40 @@
+# Node.js Google Cloud Storage sample for Google App Engine
```

Comment:
Does this need to be split into separate `standard` and `flexible` samples?

Reply:
This one is split as App Engine Node.js Std does not support the resumable upload feature in Cloud Storage client library. Should we use the sample w/ resumable upload disabled for both std and flex?

Reply:
@frankyn WDYT?